nvflare/fuel/utils/class_utils.py

```diff
@@ -85,10 +86,10 @@ def __init__(self, base_pkgs: List[str], module_names: List[str], exclude_libs=T
         self.exclude_libs = exclude_libs
 
         self._logger = logging.getLogger(self.__class__.__name__)
-        self._class_table: Dict[str, str] = {}
-        self._create_classes_table()
+        self._class_table = create_classes_table_static()
 
     def _create_classes_table(self):
```

Comment:
this method is not used anymore should we remove it?

Reply:
Keep this implementation code for referencing purpose.